.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "editor.formatOnSave": false,
+  "prettier.enable": false
+}
```

Comment:
How about `.vscode` just gets added to `.gitignore`? Not sure I want to accumulate everyone's editor files, but I'd be open to adding a single `.editorconfig`.

Reply:
I don't know enough about .editorconfig so will defer to you on this, but since the project doesn't have an automatic formatter it seems like these are wise things for anyone using VS Code to have turned off for this project. I'll read up on .editorconfig, though -- not familiar with it. And I'll add .vscode to the .gitignore.